nms/app/packages/magmalte/app/components/ActionTable.js

```diff
@@ -91,7 +91,7 @@ type ColumnType =
   | 'time'
   | 'currency';
 
-type ActionTableColumn = {
+export type ActionTableColumn = {
```

Comment:
I don't see this being used. Do we need to export this ?

Reply:
no I forgot to remove it